### DISCUSSION TITLE
Wirecutter can Break Doors

### DISCUSSION THREAD

**Discussion by Rad-Ezz** [Vdauphin/HeartsAndMinds#1646] [Question and answer] [answered]
Hi

On the new upgrade of H&M 1.24.2

FIX: Backpacks with integrated Wirecutter can Break Doors (@mrschick).

Can you explain how is work ?

**Comment by mrschick** [comment, marked as answer]
https://github.com/Vdauphin/HeartsAndMinds/pull/1644

> Allow Break Door interaction when carrying a vest or backpack defined with it, as per the ACE [wirecutter framework](https://ace3.acemod.org/wiki/framework/wirecutter-framework).

Basically, backpacks that have a wirecutter model attached, like one of the RHSAFRF's Engineer backpack variants, can now also be used to break doors, without requiring an additional wirecutter in the inventory.
